planckton/init.py

```diff
@@ -186,6 +187,39 @@ def pack(self, box_expand_factor=5):
             typed_system.strip([a.atomic_number == 1 for a in pmd_system.atoms])
         return typed_system
 
+    def stack(self, separation=0.7):
+        """Stack compounds in a single layer
+
+        Parameters
+        ----------
+        separation : float, default=0.7
+            Distance in nm to separate the individually stacked compounds
+
+        Returns
+        -------
+        typed_system : ParmEd structure
+            ParmEd structure of the box with the stacked compounds
+        """
+
+        system = mb.Compound()
+        L = self.L.value
+
+        for idx, comp in enumerate(self.compound):
+            z_axis_transform(comp)
+            comp.translate(np.array([separation, 0, 0]) * idx)
+            system.add(comp)
+
+        system.box = mb.box.Box([L, L, L])
+
+        system.translate_to(
+            (system.box.Lx / 2, system.box.Ly / 2, system.box.Lz / 2)
+        )
+
+        system.to_parmed()
```

Comment:
Check out where this same line is used in the `pack` function. Pass in the `residues=[self.residues]` here as well.